docs/HowToUseIoTjs.md

```diff
@@ -0,0 +1,63 @@
+# How to use IoT.js
+
+
+IoT.js is an open source software platform
+for Internet of Things with JavaScript.
+
+More mode detailed information, please check:
+http://www.iotjs.net
+
+Or shipped [README](..//external/iotjs/README.md) in sources.
+
+
+## Use reference configuration
+
+   machine=artik053 # Adapt with your supported target
```

Comment:
If you want to change a line, you must add two spaces at end of line.

Reply:
ok but why ?